go/stellar/format.go

```diff
@@ -142,7 +142,15 @@ func FormatAssetIssuerString(asset stellar1.Asset) string {
 	if asset.VerifiedDomain != "" {
 		return asset.VerifiedDomain
 	}
-	return "Unknown issuer"
+	iaid := asset.IssuerString()
+	iaidLen := len(iaid)
+	if iaidLen > 16 {
+		return iaid[:8] + "..." + iaid[iaidLen-8:]
```

Comment:
I really think this shortening ought to happen in the ui. Because any time the shortened version is present it should be possible to tap, expand, and copy the full issuer

Reply:
we have this in several places and I've fixed them in the UI. but in this case it's a strin gincluding the issuer description from the service, and GUI just display the string